reconcile/dashdotdb_slo.py

```diff
@@ -54,8 +61,16 @@ def dashdot_payload(self) -> dict[str, Any]:
 
 
 class DashdotdbSLO(DashdotdbBase):
-    def __init__(self, dry_run: bool, thread_pool_size: int):
-        super().__init__(dry_run, thread_pool_size, "DDDB_SLO:", "serviceslometrics")
+    def __init__(
+        self, dry_run: bool, thread_pool_size: int, secret_reader: SecretReaderBase
+    ):
```

Comment:
`-> None` missing

Reply:
Funny that mypy doesn't show that.

Reply:
Seems that `__init__` functions are treated differently by mypy https://stackoverflow.com/a/73334605/4478420

Reply:
Fixed :+1:

Reply:
Good to know. Thx